sdk/compute/armcompute/autorest.md

```diff
@@ -0,0 +1,9 @@
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+- https://github.com/Azure/azure-rest-api-specs/blob/d7dc5fbb25ed5cc718f00afb568251294c8646b4/specification/compute/resource-manager/readme.md
+- https://github.com/Azure/azure-rest-api-specs/blob/d7dc5fbb25ed5cc718f00afb568251294c8646b4/specification/compute/resource-manager/readme.go.md
```

Comment:
I have to also require the `readme.go.md` otherwise the `directive` written in `readme.go.md` will not be included in this generation